client/src/components/Categories.js

```diff
@@ -4,8 +4,9 @@ import Sewing from "../assets/sewing-machine.svg";
 import Macrame from "../assets/macrame.svg";
 import Paint from "../assets/color-palette.svg";
 import Woodwork from "../assets/woodworking.svg";
+import { Link } from "react-router-dom";
 
-const CategoryButton = styled.button`
+const NavLink = styled(Link)`
```

Comment:
There is a lot positioning going on here, maybe you can simplify that with grid, place-items, place-content or other ways? Hard to vision without a live preview 😄

Reply:
i deployed the app on heroku, link is at the end of my PR :)

Reply:
Ok I tried a little bit 😅 

<img width="482" alt="Screenshot 2020-11-26 at 11 17 58" src="https://user-images.githubusercontent.com/71655386/100338652-2760dc00-2fd9-11eb-96a5-a1c1fb3608de.png">

<img width="478" alt="Screenshot 2020-11-26 at 11 18 08" src="https://user-images.githubusercontent.com/71655386/100338661-2af46300-2fd9-11eb-95d7-e26ab321bcb1.png">

<img width="401" alt="Screenshot 2020-11-26 at 11 18 13" src="https://user-images.githubusercontent.com/71655386/100338674-2e87ea00-2fd9-11eb-88a2-dac50f3494ff.png">